applications/ConvectionDiffusionApplication/tests/thermo_mechanical_tests/coupled_problem_test_parameters.json

```diff
@@ -27,7 +27,6 @@
             "time_stepping"                   : {
                 "time_step" : 0.4
             },
-            "time_integration_method"         : "implicit",
```

Comment:
This is not the `time_integration_method` for the mechanical problem but of the conv-diff one. Please keep it. In this case I'd keep the warning to clearly inform the user of which integration method is used by default.

Reply:
I was getting an error in the CI with this. This is why I removed it. What you propose is to revert the last commit "Revert flag" 
d533805 ?

Reply:
Let me push again so we can see the error

Reply:
It seems that you have an error in the json sintax